changelog.d/12619.feature

```diff
@@ -0,0 +1 @@
+Add new `enable_registration_token_3pid_bypass` configuration option to specify a different trial period for users registered via an appservice.
```

Comment:
Just wondering why this says it is adding a `enable_registration_token_3pid_bypass` option but the code specifies a `mau_appservice_trial_days` option instead?